rre-dataset-generator/src/writers/writer_factory.py

```diff
@@ -15,7 +17,7 @@ class WriterFactory:
     }
 
     @classmethod
-    def build(cls, output_format: str) -> AbstractWriter:
+    def build(cls, output_format: str) -> Any:
```

Comment:
Why `any`? Is it because this method doesn't return the object `AbstractWriter` but an instance of it? I saw you previously used Type[class name]; it probably works here too